bsp4j/src/main/java/ch/epfl/scala/bsp4j/MavenExtension.xtend

```diff
@@ -0,0 +1,35 @@
+package ch.epfl.scala.bsp4j
+
+import java.util.List
+import org.eclipse.lsp4j.jsonrpc.validation.NonNull
+import org.eclipse.lsp4j.generator.JsonRpcData
+
+@JsonRpcData
+class MavenDependencyModule {
+  @NonNull String organization
+  @NonNull String name
+  @NonNull String version
+  @NonNull List<MavenDependencyModuleArtifact> artifacts
```

Comment:
For completeness we should have `scope` here

Reply:
Ough! Thx - fixed!